bytecode/instr.py

```diff
@@ -313,9 +314,12 @@ def stack_effect(self, jump=None):
         else:
             return dis.stack_effect(self._opcode, arg, jump=jump)
 
-    def pre_and_post_stack_effect(self, jump=None):
-
+    def pre_and_post_stack_effect(self, jump=None, check_pre_and_post=True):
```

Comment:
I dislike the idea of this function not doing what its name suggests see my above suggestion.